packages/terra-layout/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Layout
+
+The Layout component provides a responsive starting point for the positioning of application components.
```

Comment:
There was mention that were could be several layouts. Will terra-layout handles these variants or would they be separate packages? If the second, might be good to update this description to describe the layout this package provides.

Reply:
I'd assume they would all be different packages, so I can try to come up with a better description.